content/webapp/components/RelatedWorks.tsx

```diff
@@ -0,0 +1,65 @@
+import { FunctionComponent, useContext, useEffect, useState } from 'react';
+
+import { ServerDataContext } from '@weco/common/server-data/Context';
+import { Container } from '@weco/common/views/components/styled/Container';
+import Space from '@weco/common/views/components/styled/Space';
+import WorksSearchResult from '@weco/content/components/WorksSearchResult'; // temporary
+import { catalogueQuery } from '@weco/content/services/wellcome/catalogue';
+import {
+  toWorkBasic,
+  Work,
+  WorkBasic,
+} from '@weco/content/services/wellcome/catalogue/types';
+
+type Props = {
+  work: Work;
+};
+
+const RelatedWorks: FunctionComponent<Props> = ({ work }) => {
+  const [relatedContent, setRelatedContent] = useState<WorkBasic[]>([]);
+  const subjects = work.subjects.map(subject => subject.label);
```

Comment:
suggestion to clarify what the const holds: rename to `subjectsLabels`

Reply:
👍 I'll pick it up in the next iteration